specification/network/resource-manager/Microsoft.Network/stable/2022-05-01/virtualWan.json

```diff
@@ -5683,6 +5917,126 @@
         }
       }
     },
+    "subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualHubs/{virtualHubName}/inboundRoutes": {
```

Comment:
>inboundRoutes [](http://example.com/codeflow?start=128&length=13)

If you want these POST actions to be usable in an ARM template they need to be prefixed with list* 

Examples :[https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions-resource#list](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions-resource#list)

Reply:
We will resolve this in future api releases.

Reply:
That would be considered to be a breaking change and would have to to go through the whole decommisioning process if you do it later. This is a simple rename so I would suggest to fix it now.